data/json/vehicle_parts.json

```diff
@@ -5104,5 +5104,81 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "count": [1, 2]}
         ]
+    },{
+        "type" : "vehicle_part",
+        "id" : "plow",
+        "name" : "Plow",
+        "symbol" : "&",
+        "color" : "blue",
+        "broken_symbol" : "*",
+        "broken_color" : "light_blue",
+        "damage_modifier" : 40,
+        "durability" : 200,
+        "item" : "sheet_metal",
+        "difficulty" : 4,
+        "location" : "under",
+        "bonus": 150,
```

Comment:
Looks like no, I set bonus to 150,000 and it's still letting me plow with a bicycle.
You need to find another mechanism to slow the vehicle down.